arangod/Transaction/Methods.cpp

```diff
@@ -3399,22 +3361,30 @@ Result Methods::replicateOperations(LogicalCollection const& collection,
     return res;
   }
 
-  auto body = std::make_shared<std::string>();
-  *body = payload->slice().toJson();
-
   // Now prepare the requests:
-  std::vector<ClusterCommRequest> requests;
-  requests.reserve(followers->size());
-
+//  std::vector<ClusterCommRequest> requests;
+//  requests.reserve(followers->size());
+  
+  std::vector<Future<network::Response>> futures;
```

Comment:
we can consider re-adding `requests.reserve(followers->size());` as a micro optimization here.